index.html

```diff
@@ -853,66 +868,227 @@ <h2 id="BaseAudioContext">
+                <a>AudioContext</a>.
+              </li>
+              <li>Return <em>promise</em>.
+              </li>
+            </ol>
+            <p>
+              Running a <a>control message</a> to suspend an
+              <a>AudioContext</a> means running these steps on the <a>rendering
+              thread</a>:
             </p>
+            <ol>
+              <li>Attempt to <a>release system resources</a>.
+              </li>
+              <li>Queue a task on the <a>control thread</a>'s event loop, to
+              execute these steps:
+                <ol>
+                  <li>Resolve <em>promise</em>.
+                  </li>
+                  <li>If the <a href="#widl-audiocontext-state">state</a>
+                  attribute of the <a>AudioContext</a> is not already
+                  <code>suspended</code>:
+                    <ol>
+                      <li>Set the <a href="#widl-audiocontext-state">state</a>
+                      attribute of the <a>AudioContext</a> to
+                      <code>suspended</code>.
+                      </li>
+                      <li>Queue a task to fire a simple event named
+                      <code>statechanged</code> at the <a>AudioContext</a>
+                      </li>
+                    </ol>
+                  </li>
+                </ol>
+              </li>
+            </ol>
             <p>
-              While the system is suspended, MediaStreams will have their
-              output ignored; that is, data will be lost by the real time
-              nature of media streams. HTMLMediaElements will similarly have
-              their output ignored until the system is resumed. Audio Workers
-              and ScriptProcessorNodes will simply not fire their
-              onaudioprocess events while suspended, but will resume when
-              resumed. For the purpose of AnalyserNode window functions, the
-              data is considered as a continuous stream - i.e. the
-              resume()/suspend() does not cause silence to appear in the
-              AnalyserNode's stream of data.
+              While an <a>AudioContext</a> is suspended,
+              <code>MediaStream</code>s will have their output ignored; that
+              is, data will be lost by the real time nature of media streams.
+              <code>HTMLMediaElement</code>s will similarly have their output
+              ignored until the system is resumed. <a>AudioWorker</a>s and
+              <a>ScriptProcessorNode</a>s will simply not fire their
+              <code>onaudioprocess</code> events while suspended, but will
+              resume when resumed. For the purpose of <a>AnalyserNode</a>
+              window functions, the data is considered as a continuous stream -
+              i.e. the <code>resume()</code>/<code>suspend()</code> does not
+              cause silence to appear in the <a>AnalyserNode</a>'s stream of
+              data.
```

Comment:
I think the behavior of the analyser node needs to be expanded on.  If the context is suspended and I keep calling analyser node, do I get exactly the same data until the context is resumed? I think that makes the most sense